editor/import/resource_importer_scene.cpp

```diff
@@ -648,11 +648,14 @@ void ResourceImporterScene::_create_clips(Node *scene, const Array &p_clips, boo
 	AnimationPlayer *anim = Object::cast_to<AnimationPlayer>(n);
 	ERR_FAIL_COND(!anim);
 
-	if (!anim->has_animation("default")) {
+	List<StringName> default_animations;
+	anim->get_animation_list(&default_animations);
+
+	if (default_animations.size() <= 0) {
```

Comment:
> if (!default_animations.size()) {

The range of size() is 0 to the maximum int

Reply:
Thank you